lib/pxu_auth0/auth0_api.ex

```diff
@@ -4,14 +4,14 @@ defmodule PxUAuth0.Auth0API do
 
   def fetch_token(request_body),
     do:
-      post!("https://#{config(:domain)}/oauth/token", request_body, [
+      post("https://#{config(:domain)}/oauth/token", request_body, [
         {"Accept", "application/json"},
         {"Content-Type", "application/json"}
       ])
 
   def process_request_body(body), do: Jason.encode!(body)
 
-  def process_response_body(body), do: body |> Jason.decode!() |> Map.get(:body, %{})
+  def process_response_body(body), do: body |> Jason.decode!()
```

Comment:
This will raise an exception if it can't parse here, is that what we want?

Reply:
Hmmm, probably not. I think it would be better to have this bubble up an error.